examples/accessible.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: example.html
-title: Accessibility example
+title: Acessible Map
```

Comment:
Thanks for catching this @marcjansen.  I've addressed it in 19fe73e78e136fc28d38e534786e26f62804f595.